docker/main/rootfs/usr/local/nginx/templates/base_path.gotmpl

```diff
@@ -0,0 +1,17 @@
+{{ if .base_path }}
+location = {{ .base_path }} {
+    return 302 {{ .base_path }}/;
+}
+
+location {{ .base_path }}/ {
+  proxy_http_version 1.1;
+  proxy_cache_bypass $http_upgrade;
+  proxy_set_header Upgrade $http_upgrade;
+  proxy_set_header Connection $http_connection;
+  proxy_set_header Host $http_host;
+  proxy_set_header X-Ingress-Path {{ .base_path }};
+  proxy_set_header Authorization "";
+  proxy_pass http:/{{ .base_path }}:5000/;
```

Comment:
This proxies to the unauthenticated port which will be removed in a future version. We would want this to be 8971 which will get complicated since the user may or may not have TLS enabled.

```suggestion
  proxy_pass http://127.0.0.1:8971/;
```